src/transformers/keras_callbacks.py

```diff
@@ -23,6 +24,7 @@ def __init__(
         tokenizer: Optional[PreTrainedTokenizerBase] = None,
         hub_model_id: Optional[str] = None,
         hub_token: Optional[str] = None,
+        checkpoint: Optional[bool] = False,
```

Comment:
```suggestion
        checkpoint: bool = False,
```
It's only `Optional` if it can be `None`.